assets/js/base/hooks/test/use-shipping-rates.js

```diff
@@ -19,11 +19,13 @@ describe( 'useShippingRates', () => {
 	let registry, mocks, renderer;
 	const getProps = ( testRenderer ) => {
 		const {
-			shippingRates,
```

Comment:
Couldn't we just leave the `shippingRates` in the tests? Or you removed them for some reason?

Reply:
well at first I did because I also removed shippingRates from useShipping for some reason? and returned it. refactored